pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/TableRebalancerAdminToolClusterIntegrationTest.java

```diff
@@ -67,6 +67,7 @@
   private static final int NUM_INITIAL_SERVERS = 3;
   private SegmentStateTransitionStats _segmentStateTransitionStats;
   private boolean _raiseErrorOnSegmentStateTransition = false;
+  private final boolean _checkStateTransitionStats = false;
```

Comment:
Don't add this boolean, just comment out the lines and add TODO to add them back in the next pr

Reply:
Done